src/main/java/com/fitwsarah/fitwsarah/authenticationsubdomain/presentationlayer/SecurityConfig.java

```diff
@@ -54,15 +71,18 @@ public CorsConfigurationSource corsConfigurationSource() {
         return source;
     }
 
+
+
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http
                 .cors(cors -> cors.configurationSource(corsConfigurationSource()))
                 .authorizeHttpRequests(authz -> authz
+                        .requestMatchers("api/v1/appointments").hasRole("Admin")
```

Comment:
Why is the appointments end point linked to the Admin role specifically?

Reply:
Just to have an example if you want to have multiple role you can do .hasAnyRole("role1","role2")